db/seeds.rb

```diff
@@ -1,17 +1,73 @@
-user = ::Account::User.create(
+# Users
+
+joanin = ::Account::User.create!(
   email: 'joanin@katuma.org',
   first_name: 'Joanin',
   last_name: 'Cerea',
   username: 'joanin',
   password: 'kakaka22'
```

Comment:
I'm having problems getting used to this `k` thing. Can we change by `p`. Seriously is annoying :)